pkg/features/kube_features.go

```diff
@@ -302,7 +302,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	HugePages:                                   {Default: true, PreRelease: utilfeature.Beta},
 	DebugContainers:                             {Default: false, PreRelease: utilfeature.Alpha},
 	PodShareProcessNamespace:                    {Default: false, PreRelease: utilfeature.Alpha},
-	PodPriority:                                 {Default: false, PreRelease: utilfeature.Alpha},
```

Comment:
@aveshagarwal are the quota controls in place to allow this to be enabled by default?

Reply:
@liggitt I believe you are talking about https://github.com/kubernetes/kubernetes/pull/57963, this got an lgtm but waiting on approval from @bsalamat

Reply:
yes, that. until that is available, we should not enable the feature.

Reply:
Ok, I will make this alpha so that this PR is go ahead.

Reply:
Done. PTAL